### DISCUSSION TITLE
Waylock doesn't work (anymore?)

### DISCUSSION THREAD

**Discussion by bodograumann** [qtile/qtile#4158] [Q&A] [answered]
So waylock is screen locker I would like to use.
I am confused about whether it actually should be working.
I'm pretty sure it was actually working for me last fall, but now it doesn't anymore.
(So that should have been waylock 0.4.2 I guess)
The error is: `error: ext_session_lock_manager_v1 not advertised`

Seems the current version also needs the `single-pixel-buffer-v1` protocoll.

So should it be working? Is it working for you?
What alternatives are you using with wayland?

**Comment by m-col** [comment, marked as answer]
From [their release page](https://github.com/ifreund/waylock/releases) it says their 0.4.0 release depended on the `ext-session-lock-v1` protocol, which Qtile doesn't yet support unfortunately. After we move to wlroots 0.16, the next things on the wayland roadmap would be supporting some of the newer protocols such as this one.

**Reply by bodograumann** [reply]
Follow-up: swaylock still supports the older protocol beside the new. Using that going forward.

**Reply by moabeat-berlin** [reply]
With their new release 1.8.0 this also doesn't work anymore. The support for layer shell [got removed](https://github.com/swaywm/swaylock/commit/ac3b49b6571ceda3f8db11a98bfe320106996280).

**Reply by akaWolf** [reply]
@m-col and how things are going on with the supporting of ext-session-lock-v1?

**Comment by v3ss0n** [comment]
It dosen't work anymore and qtile matianers aren't very active replying.

**Comment by da-raf** [comment]
Just found this PR https://github.com/qtile/qtile/pull/5016, which seems to implement the required protocol. It requires review and testing.
